### ISSUE TITLE
Product Elements: Adding Justify content & Vertical alignment toolbar to Add to cart button

### ISSUE THREAD

**Issue by imanish003** [woocommerce/woocommerce-gutenberg-products-block#8859]
We want to add Justify content & Vertical alignment toolbar to Add to cart button, which is there in Core Button. Here is how this toolbar works in Core button.

https://user-images.githubusercontent.com/16707866/227855268-4ce118e8-781c-46af-bfc8-fdf2b5b0705e.mov

There is some discussion going on in P2(per0F9-sI-p2) to find best way to implement a Justify Content and Vertical Alignment toolbar in a block. This issue is blocked until a decision is made in P2.

This was part of #8085, but I am creating this new issue to unblock #8085.

**Comment by imanish003** [comment]
As we discussed in this discussion(per0F9-sI-p2#comment-547), I am closing this issue.

It was concluded that the existing alignment tool for the button serves the same purpose, and layout controls are not required.
